javascript/src/main/components/Statistics/DepartmentFormSelect.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { Form, Button } from "react-bootstrap";
+// hard coded depts from mongodb
+// can be dynamically fetched!
+const depts = [
+    "ANTH ",
+    "ART  ",
+    "ARTHI",
+    "ARTST",
+    "ASAM ",
+    "BLKST",
+    "BMSE ",
+    "CHEM ",
+    "CHST ",
+    "CLASS",
+    "CLIT ",
+    "CMPSC",
+    "CNCSP",
+    "CNENG",
+    "COMM ",
+    "CRSTU",
+    "EALCS",
+    "ECE  ",
+    "ECON ",
+    "EDUC ",
+    "EEMB ",
+    "ENGL ",
+    "ENGSC",
+    "ENVST",
+    "ERTSC",
+    "ESM  ",
+    "ESS  ",
+    "FEMST",
+    "FLMST",
+    "FR&IT",
+    "GEOG ",
+    "GERSL",
+    "GLOBL",
+    "GRDIV",
+    "HIST ",
+    "INT  ",
+    "L&S  ",
+    "LAIS ",
+    "LAWSO",
+    "LING ",
+    "MARSC",
+    "MAT  ",
+    "MATH ",
+    "MATRL",
+    "MCDB ",
+    "MDVST",
+    "ME   ",
+    "MILSC",
+    "MUSIC",
+    "PHIL ",
+    "PHYS ",
+    "POLS ",
+    "PSY  ",
+    "RELST",
+    "RENST",
+    "SOC  ",
+    "SP&PT",
+    "SPCH ",
+    "STATS",
+    "THTDA",
+    "TMP  ",
+    "WMST ",
+    "WRIT "
+];
```

Comment:
I added an issue for the W21 team to replace this with an API call.  It's fine that it's hard coded for now.